src/main/java/io/nats/client/Options.java

```diff
@@ -544,6 +546,10 @@ public class Options {
     private final ExecutorService executor;
     private final ServerListProvider serverListProvider;
 
+    // TODO SFF What is this

```

Comment:
We needed to add special headers when connecting through a WAF (Web Application Firewall). By supporting a list of interceptors like this, one can inject these sort of custom headers.